client/src/components/teacher-form/InputDateForm.tsx

```diff
@@ -151,12 +183,18 @@ export const InputDateForm = () => {
           <div key={i}>
             {date.startTime + ' - ' + 
             date.endTime + ',  ' +
-            date.weekDay}
+            date.dayOfWeek}
 
             <button onClick={e => deleteTerm(e, date)}
-              style={{padding: "6px 12px", margin: "10px"}}>
+              style={{padding: "6px 12px", margin: "10px 10px 10px 20px"}}>
               Usuń
             </button>
+
+            <button onClick={e => editTerm(e, date)}
```

Comment:
Wydaje mi sie ze ten komponent InputDateForm  juz za bardzo sie rozrosl i fajnie byloby troche podzielic go,
np. select dni tygodnia i godzin do nowego komponentu, button na edycje, usuwanie

Reply:
To juz jest kwestia czytelnosci, bo nawet trudniej juz mi bylo scrollowac sie po tym pliku zeby zrobic review

Reply:
Raczej już nic więcej tutaj nie dojdzie, więc chyba szkoda czasu na przerabianie tego.

Reply:
ok jasne rozumiem, czas jest naszym kluczowym aspektem XD to mozesz potraktowac jako sugestie na przyszlosc